products/pluto/pluto_build/07_custom_qaqc.sh

```diff
@@ -16,7 +16,7 @@ echo "Build staging tables"
 dbt build --select staging --exclude tag:de_check
 
 echo "Build intermediate QAQC tables"
-dbt run --select qaqc --exclude tag:de_check  # need to use 'dbt run' because otherwise it fails if downstream test fails
+dbt build --select qaqc --exclude tag:de_check
 
 echo "🔥 Run DE aka important tests 🔥"
-dbt test --select tag:de_check
+dbt test --select tag:de_check,tag:$VERSION_TYPE    # this will only run tests that have both tags, not just one of them
```

Comment:
This line will run tests that have both `de_check` and `<VERSION_TYPE>` tags. 

Why not just label tests with `<VERSION_TYPE>` tag? I was thinking about this for a while. As we create more tests, I imagine that there will be version type specific tests but not necessarily for the DE to address in case of a failure. `de_check` tag serves as a classifier: if a test fails, DE must address it. Then there will be tests/reports for GIS or tests that yield a warning. 

Would love to get some thoughts on this

Reply:
Note to self: [link](https://discourse.getdbt.com/t/exclude-multiple-tags/6538) showing how to run tests having or not having both tags (not just one of them).